jscomp/others/dom_formData.ml

```diff
@@ -0,0 +1,44 @@
+(* Provides a way to easily construct a set of key/value pairs representing form fields and their values, which can then be easily sent using the XMLHttpRequest.send() method. It uses the same format a form would use if the encoding type were set to "multipart/form-data". *)
+
+type file
+type blob
+type entryValue
+
+let classify : entryValue -> [> `String of string | `File of file ] =
+  fun t ->
+  if Js.typeof t = "string" then `String (Obj.magic t)
+  else `File (Obj.magic t)
```

Comment:
I'm not really happy with the classify, but I haven't seen any other way to achieve support for [File](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.file.html)

Reply:
Can you show me examples of why `classify` is needed?

Reply:
I'm reading it in https://github.com/reasonml/reason-react/pull/846, but I wonder whether this is needed at all or if folks can classify it downstream, or even use `Js.Types.classify`.

Besides, if we were to keep this, I'd rather call it [`Blob`](https://developer.mozilla.org/en-US/docs/Web/API/Blob) than `File`, since `File` is a subclass of `Blob`.

Reply:
Totally, the discussion for 0 runtime goes away if we remove this. If it's needed can be added in webapi

Reply:
it's a bit weird that entryValue is provided by this module but unnaccessible from this module, but it's simpler to keep it like this